src/python/py_generate_pipeline.cpp

```diff
@@ -7,11 +7,15 @@
 #include <pybind11/stl_bind.h>
 #include <pybind11/functional.h>
 #include "openvino/genai/llm_pipeline.hpp"
+#include <openvino/runtime/auto/properties.hpp>
 
 namespace py = pybind11;
 using ov::genai::LLMPipeline;
 using ov::genai::Tokenizer;
+using ov::genai::TokenizedInputs;
 using ov::genai::GenerationConfig;
+using ov::genai::EncodedInputs;
```

Comment:
How do you choose where to insert these `using`s? :) A good approach is to keep them in alphabetical order or append to the end

Reply:
sorted alphabetically